drivers/cc110x/gnrc_cc110x.c

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (C) 2016-17 Freie Universität Berlin
+ *               2018 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_gnrc_netif
+ * @{
+ *
+ * @file
+ *
+ * @author      Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ *
+ * Adapted from @ref gnrc_cc110x.c with only minor modifications. So all credit
+ * goes to Martine Lenders <m.lenders@fu-berlin.de> and to
+ * Hauke Petersen <hauke.petersen@fu-berlin.de>.
```

Comment:
Then put us in the `@author` tag ;-)

```
  * @author      Martine Lenders <m.lenders@fu-berlin.de>
  * @author      Marian Buschsieweke <marian.buschsieweke@ovgu.de>
  * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>.

Reply:
I will do that. (I interpret `@author` as "Who to ask questions about a particular file", so I hesitate to add someone without explicit permission.)

Reply:
Was moved to PR https://github.com/RIOT-OS/RIOT/pull/10638 and fixed there